monai/transforms/spatial/functional.py

```diff
@@ -610,3 +611,68 @@ def affine_func(
         out = _maybe_new_metatensor(img, dtype=torch.float32, device=resampler.device)
     out = out.copy_meta_from(meta_info) if isinstance(out, MetaTensor) else out
     return out if image_only else (out, affine)
+
+
+def convert_box_to_points(bbox, mode):
+    """
+    Convert bounding box to points.
+
+    Args:
+        mode: The mode specifying how to interpret the bounding box.
+        bbox: Bounding box in the form of [x1, y1, x2, y2] for 2D or [x1, y1, z1, x2, y2, z2] for 3D.
+            Return shape will be (N, 4) for 2D or (N, 6) for 3D.
+
+    Returns:
+        sequence of points representing the corners of the bounding box.
+    """
```

Comment:
```suggestion
    """
    Converts an axis-aligned bounding box to points.

    Args:
        mode: The mode specifying how to interpret the bounding box.
        bbox: Bounding boxes of the shape (N, C) for N boxes. C is [x1, y1, x2, y2] for 2D or [x1, y1, z1, x2, y2, z2] for 3D for each box.
            Return shape will be (N, 4, 2) for 2D or (N, 6, 3) for 3D.

    Returns:
        sequence of points representing the corners of the bounding box.
    """
```
I think this clarifies the dimensions correctly? Please check the shapes I've given and line length.